includes/admin/downloads/metabox.php

```diff
@@ -434,44 +449,48 @@ function edd_render_price_row( $key, $args = array(), $post_id, $index ) {
 
 	<div class="edd-repeatable-row-standard-fields">
 
-		<div class="edd-option-name">
-			<span class="edd-repeatable-row-setting-label"><?php _e( 'Option Name', 'easy-digital-downloads' ); ?></span>
+		<div class="edd-form-group edd-option-name">
+			<label for="edd_variable_prices-<?php echo esc_attr( $key ); ?>-name" class="edd-form-group__label edd-repeatable-row-setting-label"><?php esc_html_e( 'Option Name', 'easy-digital-downloads' ); ?></label>
+			<div class="edd-form-group__control">
 			<?php echo EDD()->html->text( array(
 				'name'        => 'edd_variable_prices[' . $key . '][name]',
+				'id'          => 'edd_variable_prices-' . $key . '-name',
 				'value'       => esc_attr( $args['name'] ),
 				'placeholder' => __( 'Option Name', 'easy-digital-downloads' ),
 				'class'       => 'edd_variable_prices_name large-text'
 			) ); ?>
+			</div>
 		</div>
 
-		<div class="edd-option-price">
-			<span class="edd-repeatable-row-setting-label"><?php _e( 'Price', 'easy-digital-downloads' ); ?></span>
+		<div class="edd-form-group edd-option-price">
+			<label for="edd_variable_prices-<?php echo esc_attr( $key ); ?>-amount" class="edd-repeatable-row-setting-label"><?php esc_html_e( 'Price', 'easy-digital-downloads' ); ?></label>
 			<?php
 			$price_args = array(
 				'name'        => 'edd_variable_prices[' . $key . '][amount]',
+				'id'          => 'edd_variable_prices-' . $key . '-amount',
 				'value'       => $args['amount'],
 				'placeholder' => edd_format_amount( 9.99 ),
-				'class'       => 'edd-price-field'
+				'class'       => 'edd-form-group__input edd-price-field',
 			);
 			?>
 
-			<span class="edd-price-input-group">
+			<div class="edd-form-group__control edd-price-input-group">
 				<?php if ( $currency_position == 'before' ) : ?>
 					<span><?php echo edd_currency_filter( '' ); ?></span>
 					<?php echo EDD()->html->text( $price_args ); ?>
 				<?php else : ?>
 					<?php echo EDD()->html->text( $price_args ); ?>
 					<?php echo edd_currency_filter( '' ); ?>
 				<?php endif; ?>
```

Comment:
Should this be formatted to match this?
https://github.com/easydigitaldownloads/easy-digital-downloads/blob/cbf390861b10f6cc1708cd38e538a4c42198d6b4/includes/admin/downloads/metabox.php#L324-L333